platforms/access-bridge/src/controllers/access-controller.ts

```diff
@@ -0,0 +1,109 @@
+import { IncomingMessage, ServerResponse } from 'http';
+
+import { AccessControlPlan } from '@jwp/ott-common/types/plans.js';
+import { Viewer } from '@jwp/ott-common/types/access.js';
+
+import { ParameterInvalidError, AccessBridgeError, sendErrors, UnauthorizedError } from '../errors.js';
+import { AccessService } from '../services/access-service.js';
+import { PlansService } from '../services/plans-service.js';
+import { isValidSiteId, parseJsonBody } from '../utils.js';
+import { AccountService } from '../services/account-service.js';
+
+/**
+ * Controller class responsible for handling access-related services.
+ */
+export class AccessController {
+  private accountService: AccountService;
+  private accessService: AccessService;
+  private plansService: PlansService;
+
+  constructor() {
+    this.accountService = new AccountService();
+    this.accessService = new AccessService();
+    this.plansService = new PlansService();
+  }
+
+  /**
+   * Service handler for generating passport access tokens.
+   * @param req The HTTP request object.
+   * @param res The HTTP response object.
+   * @param params The request parameters containing site_id.
+   */
+  generatePassport = async (req: IncomingMessage, res: ServerResponse, params: { [key: string]: string }) => {
+    try {
+      if (!isValidSiteId(params.site_id)) {
+        sendErrors(res, new ParameterInvalidError({ parameterName: 'site_id' }));
+        return;
+      }
+
+      // unauthorized is default for viewers without an authorization token
+      let viewer: Viewer = { id: 'unauthorized', email: '' };
```

Comment:
Is this a standard from the Sims side or should we get a consensus on what id/email to use in this case?

Reply:
Technically isn't this "unathenticated" not "unauthorized"?

Reply:
This is something we should introduce for free and unauthenticated viewers. You are right, I would rename it to unauthenticated, but probably something we should bring up within our checkup.